picard/tagger.py

```diff
@@ -1246,12 +1246,58 @@ def sighandler(self):
         self.signalnotifier.setEnabled(True)
 
 
-def version():
-    print("%s %s %s" % (PICARD_ORG_NAME, PICARD_APP_NAME, PICARD_FANCY_VERSION_STR))
+class PicardArgs:
 
+    def __init__(self, argparse_args=None):
+        if argparse_args is not None:
+            self.load_from_argparse(argparse_args)
 
-def longversion():
-    print(versions.as_string())
+    def load_from_argparse(self, argparse_args):
```

Comment:
This method is only called from `__init__()` so the code can be moved there and this method removed.